### ISSUE TITLE
IDEA 11 Plugin : Can't find VirtualFile for /Users/user/Library/Caches/IntelliJIdea11/extLibs/JasmineAdapterForJsTestDriver

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#307]
```
What steps will reproduce the problem?
1. To resolve Jasmine dependency in testcase click alt+enter and choose Add 
Jasmine Adapter support for jsTestDriver
2. In dialog choose Install

What is the expected output? What do you see instead?
Jasmine Adapter should be added as js dependency, but exception occured

What version of the product are you using? On what operating system?
IntelliJ IDEA 11
jsTestDriver IDEA Plugin 1.3.11

Please provide any additional information below.
Error stack trace:

Error during dispatching of 
java.awt.event.MouseEvent[MOUSE_RELEASED,(524,288),absolute(879,544),button=1,mo
difiers=Button1,clickCount=1] on dialog3: Can't find VirtualFile for 
/Users/user/Library/Caches/IntelliJIdea11/extLibs/JasmineAdapterForJsTestDriver
java.lang.RuntimeException: Can't find VirtualFile for 
/Users/user/Library/Caches/IntelliJIdea11/extLibs/JasmineAdapterForJsTestDriver
    at com.google.jstestdriver.idea.assertFramework.support.ExtractDirectoryTypeManager.getExtractDirAsVirtualFile(ExtractDirectoryTypeManager.java:178)
    at com.google.jstestdriver.idea.assertFramework.support.ExtractDirectoryTypeManager.access$600(ExtractDirectoryTypeManager.java:32)
    at com.google.jstestdriver.idea.assertFramework.support.ExtractDirectoryTypeManager$3.compute(ExtractDirectoryTypeManager.java:162)
    at com.google.jstestdriver.idea.assertFramework.support.ExtractDirectoryTypeManager$3.compute(ExtractDirectoryTypeManager.java:158)
    at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:884)
    at com.google.jstestdriver.idea.assertFramework.support.ExtractDirectoryTypeManager.extractAdapterFiles(ExtractDirectoryTypeManager.java:158)
    at com.google.jstestdriver.idea.assertFramework.support.AddAdapterSupportDialog.doOKAction(AddAdapterSupportDialog.java:86)
    at com.intellij.openapi.ui.DialogWrapper$OkAction.doAction(DialogWrapper.java:1478)
    at com.intellij.openapi.ui.DialogWrapper$DialogWrapperAction.actionPerformed(DialogWrapper.java:1449)
    at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2028)
    at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2351)
    at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:387)
    at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:242)
    at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:236)
    at java.awt.Component.processMouseEvent(Component.java:6373)
    at javax.swing.JComponent.processMouseEvent(JComponent.java:3267)
    at java.awt.Component.processEvent(Component.java:6138)
    at java.awt.Container.processEvent(Container.java:2085)
    at java.awt.Component.dispatchEventImpl(Component.java:4735)
    at java.awt.Container.dispatchEventImpl(Container.java:2143)
    at java.awt.Component.dispatchEvent(Component.java:4565)
    at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4621)
    at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4282)
    at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4212)
    at java.awt.Container.dispatchEventImpl(Container.java:2129)
    at java.awt.Window.dispatchEventImpl(Window.java:2478)
    at java.awt.Component.dispatchEvent(Component.java:4565)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:679)
    at java.awt.EventQueue.access$000(EventQueue.java:85)
    at java.awt.EventQueue$1.run(EventQueue.java:638)
    at java.awt.EventQueue$1.run(EventQueue.java:636)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:98)
    at java.awt.EventQueue$2.run(EventQueue.java:652)
    at java.awt.EventQueue$2.run(EventQueue.java:650)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:649)
    at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:686)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:530)
    at com.intellij.ide.IdeEventQueue.b(IdeEventQueue.java:416)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:374)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:205)
    at java.awt.Dialog$1.run(Dialog.java:1046)
    at java.awt.Dialog$3.run(Dialog.java:1098)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.awt.Dialog.show(Dialog.java:1096)
    at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:684)
    at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:410)
    at com.intellij.openapi.ui.DialogWrapper.showAndGetOk(DialogWrapper.java:1323)
    at com.google.jstestdriver.idea.assertFramework.support.AdapterFix$1.run(AdapterFix.java:48)
    at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:333)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:677)
    at java.awt.EventQueue.access$000(EventQueue.java:85)
    at java.awt.EventQueue$1.run(EventQueue.java:638)
    at java.awt.EventQueue$1.run(EventQueue.java:636)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:647)
    at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:686)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:534)
    at com.intellij.ide.IdeEventQueue.b(IdeEventQueue.java:416)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:374)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)

```

Original issue reported on code.google.com by `andriy.g...@gmail.com` on 7 Dec 2011 at 10:13

**Comment by GoogleCodeExporter** [comment]
Original comment by `Sergey.S...@gmail.com` on 7 Dec 2011 at 6:54
- Changed state: **Accepted**
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
```
Fix version of the plugin is 1.3.12.
```

Original comment by `Sergey.S...@gmail.com` on 26 Dec 2011 at 5:29
- Changed state: **Fixed**
